README.md

```diff
@@ -1,3 +1,22 @@
-# Loop Development Kit
```

Comment:
What happened with these readme files? Did these already exist and get moved? Was this part of your merge? I assume you did not add these as part of this excel PR?

Reply:
@cshields-oliveai This is not supposed to be my part of change. It seems to be @flowermm's Gitbook changes. these changes seems to be moved but I need to double check.